app/controllers/admins/rewards_controller.rb

```diff
@@ -39,6 +39,16 @@ def update
       render :edit, notice: e.message
     end
 
+    def import; end
```

Comment:
do we need this action?

Reply:
No, we don't need this action 🙂 But maybe with this action code is more clear? What do you think about this?

Reply:
I am not sure how an action that we do not need or use makes code clear :D

Reply:
Removed 🙂